docs/api-reference/io-time-series-change-point.md

```diff
@@ -1,7 +1,10 @@
 ### Input and Output Columns
-There is only one input column and its type is <xref:System.Single>.
-This estimator adds the following output columns:
+There is only one input column and produces a column typed to a 4-element vector.
+The output vector sequentially contains alert level (non-zero value means a change point), score, p-value, and martingale value.
 
-| Output Column Name | Column Type | Description|
-| -- | -- | -- |
-| `Prediction` | 4-element vector of <xref:System.Double> | It sequentially contains alert level (non-zero value means a change point), score, p-value, and martingale value. |
+###  Estimator Characteristics 
+|  |  | 
+| -- | -- | 
+| Does this estimator need to look at the data to train its parameters? | No | 
```

Comment:
Is that correct?

Reply:
Wrong for SSA.

---
In reply to: [278258440](https://github.com/dotnet/machinelearning/pull/3546#discussion_r278258440) [](ancestors = 278258440)